data/settings_mapping.json

```diff
@@ -234,6 +234,50 @@
         }
       ]
     },
+    {
+      "name": "starting_tab",
+      "text": "Starting Inventory",
+      "app_type": "generator",
+      "sections": [
+        {
+          "name": "starting_equipment_section",
+          "text": "Starting Equipment",
+          "col_span": 2,
+          "row_span": 2,
```

Comment:
row_span is now an array to indicate the number of rows for each column view (1 column, 2 columns and 4 columns). This allows more fine grain control over the height of sections depending on the general app size. 

This needs to be changed everywhere. A value of 2 for each size would convert to:
`"row_span": [2,2,2]`
But often you want more fine grained control then that

Reply:
Incorporated in a forthcoming commit.